setup.py

```diff
@@ -103,7 +72,3 @@ def run(self):
         ],
     },
 )
-
-
-if __name__ == '__main__':
```

Comment:
Nobody imports anything from `setup.py`, no need to check whether file was imported or executed.